### PR TITLE
Add Argument copy_tags_to_snapshot for aws_rds_cluster_instance

### LINKED ISSUE #6355: Add Argument copy_tags_to_snapshot for aws_rds_cluster_instance
_This issue was originally opened by @khannavin as hashicorp/terraform#19279. It was migrated here as a result of the [provider split](https://www.hashicorp.com/blog/upcoming-provider-changes-in-terraform-0-10/). The original body of the issue is below._

<hr>

### Current Terraform Version
Terraform v0.10.8

```
...
```

### Use-cases
Currently, aws_rds_cluster_instance is not supporting copy_tags_to_snapshot argument

### Attempted Solutions
copy_tags_to_snapshot argument is already implemented under aws_db_instance
https://www.terraform.io/docs/providers/aws/r/db_instance.html#copy_tags_to_snapshot

### PR DESCRIPTION
**Pull request by kterada0509**
Fixes #6355 

Changes proposed in this pull request:

* Add `copy_tags_to_snapshot` argument
* Update document about `copy_tags_to_snapshot` argument

Output from acceptance testing:

```
$ make testacc TESTARGS='-run=TestAccAWSRDSClusterInstance_CopyTagsToSnapshot'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./... -v -parallel 20 -run=TestAccAWSRDSClusterInstance_CopyTagsToSnapshot -timeout 120m
?   	github.com/terraform-providers/terraform-provider-aws	[no test files]
=== RUN   TestAccAWSRDSClusterInstance_CopyTagsToSnapshot
=== PAUSE TestAccAWSRDSClusterInstance_CopyTagsToSnapshot
=== CONT  TestAccAWSRDSClusterInstance_CopyTagsToSnapshot
--- PASS: TestAccAWSRDSClusterInstance_CopyTagsToSnapshot (832.74s)
PASS
ok  	github.com/terraform-providers/tmp-terraform-provider-aws/aws	832.834s
```

### CODE REVIEW

File: aws/resource_aws_rds_cluster_instance_test.go [hunk 1]
```diff
@@ -52,6 +52,7 @@ func TestAccAWSRDSClusterInstance_basic(t *testing.T) {
 					testAccCheckAWSDBClusterInstanceAttributes(&v),
 					resource.TestMatchResourceAttr("aws_rds_cluster_instance.cluster_instances", "arn", regexp.MustCompile(`^arn:[^:]+:rds:[^:]+:[^:]+:db:.+`)),
 					resource.TestCheckResourceAttr("aws_rds_cluster_instance.cluster_instances", "auto_minor_version_upgrade", "true"),
+					resource.TestCheckResourceAttr("aws_rds_cluster_instance", "copy_tags_to_snapshot", "false"),
```
Comment (bflad) [inline aws/resource_aws_rds_cluster_instance_test.go]: This is missing the resource name and causes the test to fail:

```
--- FAIL: TestAccAWSRDSClusterInstance_basic (702.58s)
    testing.go:538: Step 0 error: Check failed: Check 5/11 error: Not found: aws_rds_cluster_instance in [root]
```

```suggestion
					resource.TestCheckResourceAttr("aws_rds_cluster_instance.cluster_instances", "copy_tags_to_snapshot", "false"),
```

File: aws/resource_aws_rds_cluster_instance_test.go [hunk 2]
```diff
@@ -774,3 +804,40 @@ resource "aws_rds_cluster_instance" "test" {
...
+  master_username    = "foo"
+  master_password    = "mustbeeightcharaters"
+  skip_final_snapshot = true
+}
+
+resource "aws_rds_cluster_instance" "cluster_instances" {
+  identifier              = "tf-cluster-instance-%d"
+  cluster_identifier      = "${aws_rds_cluster.default.id}"
+  instance_class          = "db.t2.small"
+  db_parameter_group_name = "${aws_db_parameter_group.bar.name}"
```
[the remark this answers is not in the document]
Comment (kterada0509) [reply aws/resource_aws_rds_cluster_instance_test.go]: removed it.

File: aws/resource_aws_rds_cluster_instance_test.go (hunk 1 above)
Comment (kterada0509) [reply aws/resource_aws_rds_cluster_instance_test.go]: thanks.
Fixed it.

**Comment by bflad** [comment]
This has been released in [version 1.51.0 of the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1510-december-05-2018). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.

**Comment by kierenscott** [comment]
I've tested this and noticed that tags are not copied to automated and manual cluster snapshots, even though I have copy_tags_to_snapshots to true in aws_rds_cluster_instance. My Cluster and Cluster Instances are tagged. Has anyone else noticed this?

**Comment by DrFaust92** [comment]
Closes #2565

**Pull request merged**